backend/src/ee/services/scim/scim-service.ts

```diff
@@ -432,17 +468,19 @@ export const scimServiceFactory = ({
 
     if (!active) {
       // tx
-      await deleteOrgMembership({
+      await deleteOrgMembershipFn({
```

Comment:
I think we should wrap this entire function in a transaction to ensure that no memberships are actually deleted if one part of the function fails

Reply:
The function actually already is a transaction; the transaction is inside of it.